test_unittest.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from input import function
+
+class Test(unittest.TestCase):
+    def test_correct(self):

```

Comment:
Similarly, these tests should have names that reflect what they're testing:
```suggestion
    def test_user_input_rejected_containing_mistake(self):
```